code/style.css

```diff
@@ -1,4 +1,310 @@
-html {
-  background: #0025ff;
-  color: #fff;
+/*MOBILE (max-width=667px) */
+
+body {
+  min-width: 500px;
+  background: white;
+  font-family: helvetica;
+  margin: 0;
 }
+
+/*Header*/
+header {
+  width: 100%;
+  background: rgba(247, 222, 0, 0.959);
+  height: 150px;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+}
+
+.hamburger {
+  display: flex;
+  width: 30px;
+  height: 30px;
+  flex-direction: column;
+  justify-content: space-between;
+  margin-left: 20px;
+}
+
+.hamburger div {
+  width: 30px;
+  height: 5px;
+  background: white;
+  border-radius: 5px;
+}
+
+h1 {
+  font-size: 50px;
+  color: white;
+  margin: 30px;
+}
+
+.logo {
+  display: flex;
+  width: 130px;
+  justify-content: center;
+  margin: 20px;
+}
+
+nav {
+  display: none;
+  margin: 0px 40px;
+}
+
+nav a {
+  display: none;
+  color: white;
+  text-decoration: none;
+  margin-right: 25px;
+  font-size: 20px;
+}
+
+/*Big News*/
+.container {
+  background: rgb(212, 83, 93);
+  padding: 40px 0px;
+  width: auto;
+  border-radius: 5px;
+  margin: 30px;
+  min-width: 400px;
```

Comment:
Same with this one, it causes a horizontal scroll on mobile (which is something we want to avoid when working with responsive web design). Go through your code in inspector mode (in mobile width) and check for other elements that are wider than they are supposed to.